app/models/concerns/labware_creators/donor_pooling_calculator.rb

```diff
@@ -226,14 +253,14 @@ def adjust_number_of_cells_per_chip_well(count_of_samples_in_pool, number_of_cel
     # calculate chip loading volume
     chip_loading_volume = calculate_chip_loading_volume(number_of_cells_per_chip_well)
 
-    # calculate full allowance
-    full_allowance = calculate_full_allowance(chip_loading_volume)
+    # calculate allowance
+    allowance = calculate_allowance(chip_loading_volume, allowance_band)
```

Comment:
```suggestion
    volume_needed = calculate_allowance(chip_loading_volume, allowance_band)
```

Just aware there's a lot of jargon - maybe we can rename some of these variables to make it more intuitive to understand?